tests/UnitTests/CoreGraphics.drawing/DrawingTest.cpp

```diff
@@ -34,10 +38,14 @@ void testing::DrawTest<TComparator>::SetUp() {
     CGSize size = CanvasSize();
 
     auto deviceColorSpace = woc::MakeStrongCF<CGColorSpaceRef>(CGColorSpaceCreateDeviceRGB());
-    _context.reset(CGBitmapContextCreate(
-        nullptr, size.width, size.height, 8, size.width * 4, deviceColorSpace, kCGImageAlphaPremultipliedFirst));
+    _context.reset(
+        CGBitmapContextCreate(nullptr, size.width, size.height, 8, size.width * 4, deviceColorSpace, kCGImageAlphaPremultipliedFirst));
     ASSERT_NE(nullptr, _context);
 
+#if WINOBJC // Validate that the results are correct even under batched drawing from _CGContextPushBegin/PopEndDraw
```

Comment:
It might even be more correct to do this in CGBitmapContext.

Here's what we can potentially do:

If the user passes `!data`, we push a begin draw. When they call CreateImage/GetImage, we pop an endDraw if we know we are in control of the data.

That way, people using `data` get immediate mode rendering, and people not using data automatically get batched mode rendering.

This opens up some testing avenues that don't require internal implementation details (pass data, do not pass data, etc.).

I do, however, think this is the right thing to do here until we normalize on a strategy for bitmap contexts.